src/renderer/i18n/de/common.ts

```diff
@@ -46,7 +46,10 @@ const common: CommonMessages = {
   'common.reload': 'Neuladen',
   'common.add': 'Einzahlen',
   'common.swap': 'Swap',
-  'common.liquidity': 'Liquidität'
+  'common.liquidity': 'Liquidität',
+  'common.validations.lessThen': 'Should be less then {value} - DE',
```

Comment:
```
  'common.validations.lessThen': 'Der eingegebene Betrag sollte weniger als {value} betragen',
```

^ Move it into `i18n/de/wallet` (same for following validation messages and for other languages)